proposals/lip-0048.md

```diff
@@ -238,7 +243,13 @@ def afterCommandExecute(trs: Transaction) -> None:
     senderAddress = sha256(trs.senderPublicKey)[:LENGTH_ADDRESS]
     # The fee paid to the generator is the difference between the fee paid by the sender and the paid fees.
     Token.unlock(senderAddress, MODULE_NAME_FEE, TOKEN_ID_FEE, trs.fee)
-    Token.burn(senderAddress, TOKEN_ID_FEE, trs.fee - ctx.availableTransactionFee)
+
+    if ADDRESS_FEE_POOL is not None:
+        # Transfer the fees to the pool account address, if it exists.
+        Token.transfer(senderAddress, ADDRESS_FEE_POOL, TOKEN_ID_FEE, trs.fee - ctx.availableTransactionFee)
+    else:
+        Token.burn(senderAddress, TOKEN_ID_FEE, trs.fee - ctx.availableTransactionFee)
+
     Token.transfer(senderAddress, generatorAddress, TOKEN_ID_FEE, ctx.availableTransactionFee)
 
     emitEvent(
```

Comment:
The term `burntAmount` is not correct anymore. I would replace it with `minimalAmount`, but maybe there are better alternatives.
For a more specific case of DEX I we use the term `sharedAmountValidators`.

Reply:
Agreed, I will change all instances of the `burntAmount` to `minimalAmount`.

Reply:
The comment is valid, but `minimalAmount` is not intuitive at all IMO, I would suggest finding something better otherwise keep the old naming.

Reply:
To me it seems that `minimalAmount` is still better than `burntAmount` since, by the modification this PR introduces, the minimal amount of the fee is not necessarily being burnt anymore. 
Maybe it can be simply `minimalFee`?

Reply:
ok in fact now that I think again of it, I see the point of using `minimalAmount` or `minimumFee`, it is the minimum required for the transaction to succeed. We could also use `baseFee`/ `baseAmount` or `requiredAmount`, meaning this is the basic fee you need for the transaction, meaning whatever is above this (ctx.availableTransactionFee``), should be treated as "optional". 

Not a big deal, but we should end up with something intuitive.

Reply:
Actually, `requiredAmount` sounds good to me, if @sergeyshemyakov agrees we can use this option.